manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -179,8 +179,11 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
       title = "Update Graph",
       text =
         """Applies updates to the graph.""".stripMargin,
-      queryText = """CREATE (:Person {name: "Alistair"})""",
-      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("CreateNode"))
+      queryText = """CYPHER planner=rule CREATE (:Person {name: "Alistair"})""",
```

Comment:
Perhaps you're right. There's a card on the Cypher wall asking the team to update the manual with all the new writing operators, but it's not going to be played any time soon. In the meantime, I think making this example anything but nonsensical would be an improvement.